.github/workflows/link-check.yml

```diff
@@ -26,7 +26,7 @@ on:
         required: true
         default: 'published'
         type: string
-      create_issue:
+      linkcheck_create_issue:
         description: 'create new GitHub issue if broken links found'
         required: false
         default: true
```

Comment:
Non-blocking: not immediately applicable to this PR and not not this block of code, this block is just the most relevant. Nikita figured out an ingenious way to keep posting to the same issue instead of spamming the repo with a new issue per failure, you want to check this out within this PR or a follow-up one. It made the team less upset in the OSCAL work. 😄 

https://github.com/usnistgov/OSCAL/pull/1916

Reply:
Discussed in chat.